video/out/vo_kitty.c

```diff
@@ -48,7 +48,7 @@
 #define DEFAULT_WIDTH 80
 #define DEFAULT_HEIGHT 25
 
-static inline void write_str(const char *s)
+static inline void write_bstr(const bstr *bs)
```

Comment:
It's fine to do a copy, bstr is pointer and len only.
```suggestion
static inline void write_bstr(bstr bs)
```